apps/admin-gui/src/app/vos/pages/group-detail-page/group-subgroups/group-subgroups.component.ts

```diff
@@ -157,4 +174,46 @@ export class GroupSubgroupsComponent implements OnInit {
       }
     });
   }
+
+  displaySelectedRoles(): string {
+    if (this.selectedRoles.length === this.rolesList.length) {
+      return 'ALL';
+    }
+    const roles: string[] = this.roles.value as string[];
+    if (roles) {
+      return `${roles[0]}  ${
+        roles.length > 1
+          ? '(+' + (roles.length - 1).toString() + ' ' + (roles.length === 2 ? 'other)' : 'others)')
+          : ''
+      }`;
+    }
+    return '';
+  }
+
+  displaySelectedRoleTypes(): string {
+    if (this.selectedRoleTypes.length === this.roleTypesList.length) {
+      return 'ALL';
+    }
+    const types: string[] = this.roleTypes.value as string[];
+    if (types) {
+      return `${types[0]}  ${
+        types.length > 1
+          ? '(+' + (types.length - 1).toString() + ' ' + (types.length === 2 ? 'other)' : 'others)')
+          : ''
+      }`;
```

Comment:
This is not needed when there are just 2 possible options, so it can be `ALL` or something like `types[0] + ' 1 other'`.